### PR TITLE
set filename for remote files to allow debugability

### PR DESCRIPTION
**Pull request by itaibh**
When loading a code file from a URL into a VM, also set its filename, so you can get back to it to debug it, and it won't get overriden by the default name (`evalmachine.<anonymous>`)

### CODE REVIEW

**Review by kyrylodolynskyi — APPROVED** [review]
LGTM! I fixed some code styles.

File: js/packages/sdk-coverage-tests/src/emitter.js [hunk 1]
```diff
@@ -145,7 +145,10 @@ function useEmitter() {
       },
     }
 
-    return new Proxy(function () {}, {
+    const base = function () {
+      return undefined
+    }
+    return new Proxy(base, {
```
Comment (amitzur) [inline js/packages/sdk-coverage-tests/src/emitter.js:151]: @itaibh @kyrylodolynskyi why was this change needed?

File: js/packages/sdk-coverage-tests/test/cli-util.spec.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ describe('cli-util', () => {
       assert.deepStrictEqual(getTestIndexesFromErrors([{}]), undefined)
     })
     it('should get test indexes for passed tests', () => {
-      const tests = {a: () => {}, b: () => {}, c: () => {}, d: () => {}}
+      const tests = {a: () => void 0, b: () => void 0, c: () => void 0, d: () => void 0}
```
Comment (amitzur) [inline js/packages/sdk-coverage-tests/test/cli-util.spec.js:19]: @itaibh @kyrylodolynskyi why was this change needed?
